cluster/gce/gci/configure-helper.sh

```diff
@@ -2686,6 +2681,16 @@ oom_score = -999
 [plugins.cri.registry.mirrors."docker.io"]
   endpoint = ["https://mirror.gcr.io","https://registry-1.docker.io"]
 EOF
+
+  # Reuse docker group for containerd.
+  local containerd_gid="$(cat /etc/group | grep ^docker: | cut -d: -f 3)"
```

Comment:
before this change the script was assuming that `docker` is always installed!

Reply:
IIRC this is in fact true on the images we use?